

#------------- Rocket parameters --------------------

#

# Defines parameters specific to the rocket:

#   - General dimensions of the components

#   - Rocket engine specifications

#

# ----------------------------------------------------

stages: 4

diameters:

  - 0.26

  - 0.26

  - 0.26

  - 0.26

stage_z:
  - 0
  - 0.2
  - 0.3
  - 0.5

fin_n: 3

fin_top_offset: 0.25

fin_span: 1.0e-40

fin_root_chord: 0.1 # base length

fin_tip_chord: 0.1

fin_sweep: 0.1

fin_thickness: 0.004

lug_n: 2

lug_S: 1.0e-40

dry_mass: 1.67 # 1.67 without cover, +0.28 = 1.95 with cover

dry_CM: 0.215

dry_I:

  - 0.0644

  - 0.0644

  - 0.0128

propellant_mass: 1.0e-99

propellant_CM: 0.1

#set Isp to -1 for constant propellant mass

Isp: 1.0e+99

Cd:

  - 2

  - 2

  - 0.7

throttling: True

input_delay: 34.0e-3 # Mean delay between external actions and its reception

output_delay: 80.0e-3 # Mean delay between output command and its effect

#max_propeller_delta: 40

max_gimbal_angle: 15

max_gimbal_rate: 300

#Adapted to drone

maxThrust:

  - 100

  - 100

  - 19.031

minThrust:

  - -100

  - -100

  - 0

minTorque: -50

maxTorque: 50

